raw_span.go

```diff
@@ -38,6 +38,9 @@ type SpanContext struct {
 	// A probabilistically unique identifier for a [multi-span] trace.
 	TraceID uint64
 
+	// Least significant bits of TraceID
+	TraceIDLower uint64
```

Comment:
Do we need to add the field yet, or can we just drop the lower bits for now? Doesn't look like we have anything consuming the new `TraceIDLower`, e.g., for propagation.

Reply:
The field gets propagated is used in the b3 propagation: https://github.com/lightstep/lightstep-tracer-go/pull/232/files#diff-c7e8819bc83013a318340945ad302ef3R36

Reply:
Doh!